.travis.yml

```diff
@@ -41,17 +38,15 @@ install:
 script:
   # Lint text-like files
   - scripts/pre-commit.sh --require-all
-  - make vendor-install
+  # Fetch vendor dependencies so linters succeed
+  - dep ensure -v -vendor-only
```

Comment:
`make vendor-install` is same but without `-v` flag, why cant we use `make vendor-install` instead of dep-command?

Reply:
I'm setting the stage for getting rid of the Makefile. It doesn't actually manage dependencies like a Makefile is designed to do. It's current usage is just a way of running scripts. Once everything is part of the container build process, the only command to run is to build the container.

The `-v` was added to send addl output to Travis, preventing timeouts in case it takes a long time to fetch the vendor libs.

Reply:
I think this is good. +1

Reply:
@JohnStrunk I am wondering why we need to get rid of `makefile` completely? Cant we keep that for some developers who dont want to do container build at time of development ? I am fine with removing the make stuff from our travis though.